src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -225,32 +235,61 @@ func (r *ClusterReconciler) createBootstrapConfigMap(
 	cluster *redpandav1alpha1.Cluster,
 	scheme *runtime.Scheme,
 ) error {
+	serviceAddress := cluster.Name + "." + cluster.Namespace + ".svc.cluster.local"
 	cfg := config.Default()
 	cfg.Redpanda = copyConfig(&cluster.Spec.Configuration, &cfg.Redpanda)
 	cfg.Redpanda.Id = 0
 	cfg.Redpanda.AdvertisedKafkaApi.Address = cluster.Name + seedSuffix + "-0" + "." +
 		cluster.Name + seedSuffix + "." +
 		cluster.Namespace +
 		".svc.cluster.local"
+	cfg.Redpanda.AdvertisedKafkaApi.Port = cfg.Redpanda.KafkaApi.Port
 	cfg.Redpanda.AdvertisedRPCAPI.Address = cluster.Name + seedSuffix + "-0" + "." +
 		cluster.Name + seedSuffix + "." +
 		cluster.Namespace +
 		".svc.cluster.local"
+	cfg.Redpanda.AdvertisedRPCAPI.Port = cfg.Redpanda.RPCServer.Port
 	cfg.Redpanda.Directory = dataDirectory
+	cfg.Redpanda.SeedServers = []config.SeedServer{
+		{
+			Host: config.SocketAddress{
+				// Example address: cluster-sample-seed-0.cluster-sample.default.svc.cluster.local
+				Address:	cluster.Name + seedSuffix + "-0." + serviceAddress,
+				Port:		cfg.Redpanda.AdvertisedRPCAPI.Port,
+			},
+			Id:	0,	// Start from 0 to match statefulset pod naming.
```

Comment:
A change went in today that eliminates the need to set the Id on seed servers altogether.

Reply:
Thanks, I removed the Id.